documentation/installation.md

```diff
@@ -7,9 +7,11 @@ Supported (continuously tested) platforms:
 
```

Comment:
The macOS version mentioned in Requirements (not changed in the pull request) is outdated.

Reply:
so which one are the new build nodes using? Catalina?

Reply:
Yes.